src/qt/walletmodel.h

```diff
@@ -59,12 +58,6 @@ class SendCoinsRecipient
     CAmount amount;
     // If from a payment request, this is used for storing the memo
     QString message;
-
-    // If from a payment request, paymentRequest.IsInitialized() will be true
-    PaymentRequestPlus paymentRequest;
-    // Empty if no authentication or invalid signature/cert/etc.
-    QString authenticatedMerchant;
```

Comment:
note: `authenticatedMerchant` was not removed in the original PR

Reply:
I think it would be to preserve the label information from previous BIP70 transactions. The only vague indication seems to be https://github.com/bitcoin/bitcoin/pull/11536.